Code/Tools/RenegadeGR/rengameres.cpp

```diff
@@ -266,24 +266,24 @@ char** RenegadeGameRes::_addToArr(char** arr, const char* item)
 | Takes a pointer to an array and a new item, constructs a new array and returns	|
 | a pointer to it.																	|
 `----------------------------------------------------------------------------------*/
-long unsigned int* RenegadeGameRes::_addToArr(long unsigned int* arr, long unsigned int item)
+int unsigned int* RenegadeGameRes::_addToArr(int unsigned int* arr, int unsigned int item)
 {
-	long unsigned int* newarr = NULL;
+	int unsigned int* newarr = NULL;
 
 
 	if( arr == NULL )
 	{
 		// Make a new array
 		assert( _myplayercount == 0 );
-		newarr = new long unsigned int[1];		
+		newarr = new int unsigned int[1];		
```

Comment:
`new int unsigned int`?  that doesn't look right!

Reply:
Thankfully we don't build this target, I'll get it sorted though.